_doc-build-linux/action.yml

```diff
@@ -119,6 +119,14 @@ inputs:
     required: true
     type: boolean
 
+  add-cheatsheet:
```

Comment:
Can we rename this to ``add-quarto-cheatsheet`` ?

Reply:
sure , i can add those.